Makefile

```diff
@@ -177,6 +177,8 @@ FLAVOR ?= e2e
 
 TEST_NAMESPACE=capx-test-ns
 TEST_CLUSTER_NAME=mycluster
+TEST_CLUSTER_CLASS_NAME=my-clusterclass
```

Comment:
Do we need a separate var here? This now requires more documentation and remembering what value to set when using the different flavors.

Reply:
this is for dev testing only, we can enhance if needed in subsequent PRs.

Reply:
Sorry, I got confused with this var. I meant to keep `TEST_CLUSTER_NAME` as the variable and remove the new `TEST_TOPOLOGY_CLUSTER_NAME` var.

As discussed on a call though, this is not blocking and can be tackled in a separate PR to rework a little the dev experience.